### ISSUE TITLE
how to optimize

### ISSUE THREAD

**Issue by goog** [jcjohnson/torch-rnn#229]
Epoch 43.28 / 50, i = 822990 / 973300, loss = 4.399446
Epoch 43.28 / 50, i = 822991 / 973300, loss = 4.486917
Epoch 43.28 / 50, i = 822992 / 973300, loss = 4.432359
Epoch 43.28 / 50, i = 822993 / 973300, loss = 4.236250
Epoch 43.28 / 50, i = 822994 / 973300, loss = 4.456953
Epoch 43.28 / 50, i = 822995 / 973300, loss = 4.398404
Epoch 43.28 / 50, i = 822996 / 973300, loss = 4.325682
Epoch 43.28 / 50, i = 822997 / 973300, loss = 4.290632
Epoch 43.28 / 50, i = 822998 / 973300, loss = 4.397292
Epoch 43.28 / 50, i = 822999 / 973300, loss = 4.272812
Epoch 43.28 / 50, i = 823000 / 973300, loss = 4.496970
